src/blocks/VideoBlock.tsx

```diff
@@ -1,102 +1,112 @@
-import Block from './Block'
-import { BlockModel } from './types'
+import React, { useState, useEffect } from 'react';
+import Block from './Block';
+import { BlockModel } from './types';
 import BlockFactory from './BlockFactory';
-import './BlockStyles.css'
-import Box from "@mui/material/Box";
-import TextField from "@mui/material/TextField";
-import Button from "@mui/material/Button";
-import Typography from "@mui/material/Typography";
-import TitleComponent from './utils/TitleComponent';
+import './BlockStyles.css';
+import ReactPlayer from 'react-player/lazy';
+import { Box, IconButton } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
 import FileUploadComponent from './utils/FileUploadComponent';
-import ReactPlayer from 'react-player/lazy'
 import SeamSaveButton from '../components/SeamSaveButton';
+import EmptyVideoIcon from '../blocks/blockIcons/EmptyVideoIcon.png';
 
 export default class VideoBlock extends Block {
   render() {
     if (Object.keys(this.model.data).length === 0) {
-      return BlockFactory.renderEmptyState(this.model, this.onEditCallback!)
+      return BlockFactory.renderEmptyState(this.model, this.onEditCallback!);
     }
 
     const url = this.model.data["url"];
-    const title = this.model.data["title"];
     if (!url) {
       return this.renderErrorState();
     }
 
     return (
-      <div style={{ backgroundColor: this.theme.palette.secondary.main, width: "100%" }}>
-        {title && TitleComponent(this.theme, title)}
+      <div style={{ backgroundColor: this.theme.palette.secondary.main }}>
         <div className="flex grow relative w-full h-full min-h-[300px]" style={{ position: 'relative', width: "100%", height: "100%" }}>
           <ReactPlayer
             controls={true}
             url={url}
-            style={{ minHeight: '300px', width: '100%', height: 'auto', flexGrow: 1, }}
+            style={{ height: '100%', width: '100%', flexGrow: 1 }}
```

Comment:
backwards compatibility, this changes the sizing for all previous video posts: http://localhost:3000/search/video

Reply:
Yes, unfortunately only for videos that can embed or were previously uploaded. (Video posts made with an instagram link for example will say unsupported MIME type)